### DISCUSSION TITLE
Multiple best values for a list of hyperparameter?

### DISCUSSION THREAD

**Discussion by praksharma** [optuna/optuna#4182] [Q&A]
I am using Optuna with Hydra.

```sh
defaults:
  - override hydra/sweeper: optuna
  - override hydra/sweeper/sampler: tpe

hydra:
  mode: MULTIRUN
  sweeper:
    sampler:
      seed: 123
    direction: minimize
    study_name: sphere
    storage: null
    n_trials: 20
    n_jobs: 1
    params:
      +n: 2,3,4
      x: range(0, 5, step=1.0)
      y: choice(-5 ,0 ,5)
      +a_lims: "{a_lower : 0.5, a_upper : 1.1, a : 0.91},{a_lower : 0.7, a_upper : 1.15, a : 1.05}, {a_lower : 0.9, a_upper : 1.15, a : 1.09}, {a_lower : 0.97, a_upper : 1.05, a : 1.01}"
x: 1
y: 1
```

Here i am trying to optimise my training loss for each combination of `n` and `a_lims`. So, if `n` can take 3 values and `a_lims` can take 4 values then I end up with 12 different best values. Here I would only like to optimise `x` and `y` for each combination of `n` and `a_lims`.

Is that possible?
